[2023-06-01T11:22:20.477+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-04-26T00:00:00+00:00 [queued]>
[2023-06-01T11:22:20.517+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-04-26T00:00:00+00:00 [queued]>
[2023-06-01T11:22:20.518+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:22:20.519+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:22:20.520+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:22:20.572+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-04-26 00:00:00+00:00
[2023-06-01T11:22:20.590+0000] {standard_task_runner.py:55} INFO - Started process 1434 to run task
[2023-06-01T11:22:20.613+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2022-04-26T00:00:00+00:00', '--job-id', '432', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmp2w0ky6bq']
[2023-06-01T11:22:20.616+0000] {standard_task_runner.py:83} INFO - Job 432: Subtask insert_into_table
[2023-06-01T11:22:20.783+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-04-26T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:22:21.036+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-04-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-26T00:00:00+00:00
[2023-06-01T11:22:21.052+0000] {sql.py:253} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-04-26', 'dag_with_postgres_operator_v03')
        
[2023-06-01T11:22:21.086+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:22:21.650+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:22:21.679+0000] {sql.py:364} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-04-26', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-06-01T11:22:21.689+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-01T11:22:21.722+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20220426T000000, start_date=20230601T112220, end_date=20230601T112221
[2023-06-01T11:22:21.794+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-01T11:22:21.913+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
